index.js

```diff
@@ -119,8 +119,9 @@ async function processEvents (events) {
 }
 
 function sendEvent (event) {
-  return SQS_CLIENT.pushAsync(QUEUES[event.notificationType], event)
-    .then(() => console.log('Sent:', event.notificationType))
+  const notificationType = JSON.parse(event.Message).notificationType
```

Comment:
Instead of doing extra work here to parse a structure that we only just stringified, we could pass the object instead and stringify it internally. Then we can access `notificationType` without needing to do a parse and, bonus side-effect, we also get to remove the duplicate code that we have separately for sendgrid and socketlabs. Make sense, or am I missing something?